Hero/Hero.swift

```diff
@@ -166,18 +175,19 @@ internal extension Hero {
     // wait for a frame if using navigation controller.
     // a bug with navigation controller. the snapshot is not captured if animating immediately
     delay(inContainerController && presenting ? 0.02 : 0) {
-      for (currentFromViews, currentToViews) in self.animatorViews {
+      self.animatorViews.forEach {
         // auto hide all animated views
-        for v in currentFromViews {
-          self.context.hide(view: v)
+        $0.0.forEach {
+          self.context.hide(view: $0)
         }
-        for v in currentToViews {
-          self.context.hide(view: v)
+        
+        $0.1.forEach {
+          self.context.hide(view: $0)
```

Comment:
@kylez420 All right. I knew what you wanted. We will not use "$ 0" and will make the source code readable and beautiful. I'll fix it now.